vaas/vaas/router/models.py

```diff
@@ -6,6 +6,25 @@
 from vaas.cluster.models import LogicalCluster
 from vaas.manager.models import Director
 
+RESPONSE_STATUS_CHICES = (
+    ('301', 'Move Pernamentlly'),
+    ('302', 'Found'),
+    ('307', 'Temporary Redirect')
+)
+
+class Rewrite(models.Model):
+    condition = models.CharField(max_length=512)
+    destination = models.CharField(max_length=512)
+    action = models.CharField(max_length=64, choices=RESPONSE_STATUS_CHICES, default='301')
+    priority = models.PositiveIntegerField()
+    persevre_query_params = models.BooleanField(default=True)
```

Comment:
Pls, check one more time if that spelling is correct

Reply:
Preserve will be correct form here. I will change it.